tcmur_cmd_handler.c

```diff
@@ -745,6 +745,18 @@ static int tcmur_cmd_handler(struct tcmu_device *dev, struct tcmulib_cmd *cmd)
 
 	track_aio_request_start(rdev);
 
+	/*
+	 * Try to passthrough the cmds to the device.
+	 */
+	if (rhandler->handle_cmd) {
+		ret = handle_passthrough(dev, cmd);
+		if (ret != TCMU_NOT_HANDLED)
```

Comment:
lxbsz, nr_threads means handler can't do its own aio, so every command is kicked off in a runner thread.

liuy, yes we did. There is a new case with Venky's aio code though and we were discussing if we want to rearrange the code to handle it, do an array, or just leave it to handlers to figure it out. We decided on the latter since it was the most simple.